File: output/deepseek_chat/avatar/direct_translation/java/python/successful_translation/codeforces_682_B.py

import sys

def main():
    n = int(sys.stdin.readline())
    lst = list(map(int, sys.stdin.readline().split()))
    lst.sort()
    c = 1
    for num in lst:
        if num >= c:
            c += 1
    print(c)

if __name__ == "__main__":
    main()

File: output/deepseek_chat/avatar/direct_translation/python/java/successful_translation/codeforces_682_B.java
import java.util.*;
import java.io.*;

public class codeforces_682_B {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        String[] s = br.readLine().split(" ");
        int[] l = new int[n];
        for (int i = 0; i < n; i++) {
            l[i] = Integer.parseInt(s[i]);
        }
        Arrays.sort(l);
        int c = 0;
        for (int i : l) {
            if (i > c) c++;
        }
        System.out.println(c + 1);
    }
}
